ui/src/data/virtual_school.js

```diff
@@ -135,4 +167,212 @@ export const allStudents = [
+    "behavior": "Sent to principal for fighting with CJ",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "In drama club and loves art",
+    "familyBackground": "Single mom, two older sisters",
+    "ses": null,
+    "academicPerformance": "C in 6th grade science, B in 6th grade science"
+  }
+].map(addFields({
+  isMursion: false,
+  cohortKey: '20160722'
+}));
+
+
+// From Virtual School Context_Update_072716
+const julyStudentsFromMursion = [
+  {
+    "id": 3001,
+    "grade": "7th grade",
+    "name": "Maria Gonzales",
+    "gender": "female",
+    "race": "Dominican",
+    "behavior": "Quiet",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Loves to draw, write, and read",
+    "familyBackground": "Lives with both parents, 2 older brothers",
+    "ses": null,
+    "academicPerformance": "A in 6th grade science, A in 6th grade math"
+  }, {
+    "id": 3002,
+    "grade": "7th grade",
+    "name": "Cindy “CJ” Harper",
+    "gender": "female",
+    "race": "Caucasian",
+    "behavior": "Sent to principal for fighting with Lucy",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Loves movies, spends lots of time with 9th grade boyfriend",
+    "familyBackground": "Mom is ill, Dad is not present, older brother is often truant from high school",
+    "ses": null,
+    "academicPerformance": "F in 6th grade science, C in 6th grade math"
+  }, {
+    "id": 3003,
+    "grade": "7th grade",
+    "name": "Kevin Jordan",
+    "gender": "male",
+    "race": "African American",
+    "behavior": null,
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Plays guitar, loves mixing music",
+    "familyBackground": "Lives with grandmother, ½ sister lives with Mom and boyfriend",
+    "ses": "Free and reduced lunch",
+    "academicPerformance": "C in 6th grade science, C in 6th grade math"
+  }, {
+    "id": 3004,
+    "grade": "7th grade",
+    "name": "Ed Lewis",
+    "gender": "male",
+    "race": "African American",
+    "behavior": null,
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Wants to be an air force pilot like uncle, plays basketball",
+    "familyBackground": "Lives with both parents, younger sister and younger twin brothers",
+    "ses": null,
+    "academicPerformance": "B in 6th grade science, B in 6th grade math"
+  }, {
+    "id": 3005,
+    "grade": "7th grade",
+    "name": "Sean McGowen",
+    "gender": "male",
+    "race": "Caucasian",
+    "behavior": "Talkative",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Boy scouts, video games, math camp",
+    "familyBackground": "Lives with both parents, only child",
+    "ses": null,
+    "academicPerformance": "B in 6th grade science, C in 6th grade math"
+  }
+].map(addFields({
+  isMursion: true,
+  cohortKey: '20160722'
+}));
+
+export const allStudents = initialStudents.concat(
+  julyStudentsFromMursion,
+  julyStudentsNotFromMursion
+);
```

Comment:
For now it shouldn't be a problem but don't some of the students from `julyStudentsNotFromMursion` overlap with `initialStudents`? The entire list of students isn't really shown anywhere right now so it doesn't really matter but if I remember correctly, the student autocomplete in #104 uses `allStudents`. This would lead to having, as an example, both Mike and Mike Bilson who are the same except for the addition of an academic performance field and a change to the family background field. Should we update the original ones instead or just leave this as is until we are ready to use them?